x/pool-incentives/types/expected_keepers.go

```diff
@@ -41,3 +46,11 @@ type DistrKeeper interface {
 	SetFeePool(ctx sdk.Context, feePool distrtypes.FeePool)
 	FundCommunityPool(ctx sdk.Context, amount sdk.Coins, sender sdk.AccAddress) error
 }
+
+type EpochKeeper interface {
+	GetEpochInfo(ctx sdk.Context, identifier string) epochstypes.EpochInfo
+}
+
+type GAMMKeeper interface {
+	GetPoolType(ctx sdk.Context, poolId uint64) (poolmanagertypes.PoolType, error)
+}
```

Comment:
Why is this a gamm keeper? Shouldn't this be poolmanager keeper?

Reply:
actually we dont even use this, i'll remove it